### PR TITLE
Strip mailto: when yanking a hint

### LINKED ISSUE #61: When copying link location for mails, `mailto:` should be removed.
In the crash dialog and on web pages.

### PR DESCRIPTION
**Pull request by blyxxyz**
Resolves #61.

### CODE REVIEW

File: qutebrowser/browser/hints.py [hunk 1]
```diff
@@ -236,6 +236,7 @@ def yank(self, url, context):
                utils.supports_selection())
 
         urlstr = url.toString(QUrl.FullyEncoded | QUrl.RemovePassword)
+        urlstr = urlstr.lstrip('mailto:')
```
Comment (The-Compiler) [inline qutebrowser/browser/hints.py]: That doesn't do the right thing - it strips any of the characters m, a, i, l, t, o and :.

It'd probably be cleaner to do this before converting the URL to a string.

Comment (blyxxyz) [inline qutebrowser/browser/hints.py]: Would something like
```python
if url.scheme() == 'mailto':
    urlstr = url.path()
else:
    ...
```
be good?

Comment (The-Compiler) [inline qutebrowser/browser/hints.py]: I'd probably just do something like:

```python
flags = QUrl.FullyEncoded | QUrl.RemovePassword
if url.scheme() == 'mailto':
    flags |= QUrl.RemoveScheme
urlstr = url.toString(flags)
```

**Comment by The-Compiler** [comment]
Can you please also add a test to `end2end/features/hints.feature`?

(also, review for your other PRs might take a bit, I'm busy with exams for the next ~3 weeks)

**Comment by blyxxyz** [comment]
I already added a test there. Do you mean a more specific one?

**Comment by The-Compiler** [comment]
Oh, I must've missed that, sorry 😄

**Comment by The-Compiler** [comment]
Thanks!

**Pull request merged**